docs/DeveloperGuide.md

```diff
@@ -1,13 +1,39 @@
 # Developer Guide
```

Comment:
It might be better to include a short instruction that other developers may understand what the application will be about.